Doc/whatsnew/3.8.rst

```diff
@@ -237,6 +237,37 @@ asyncio
 
 On Windows, the default event loop is now :class:`~asyncio.ProactorEventLoop`.
 
+builtins
+--------
+
+The :func:`compile` built-in has been improved to accept the
+``ast.PyCF_ALLOW_TOP_LEVEL_AWAIT`` flag. With this new flag passed,
+:func:`compile` will allow top-level ``await``, ``async for`` and ``async with``
+constructs that are usually considered invalid syntax. Asynchronous code object
+marked with the ``CO_COROUTINE`` flag may then be returned. Those code-objects
```

Comment:
I'd drop it after "Those code-objects". It's a bit too much detail for a what's new entry.

Reply:
Done.